version.properties

```diff
@@ -1,2 +1,8 @@
-mockito.testng.version=1.0
+#Currently building Mockito version
 version=2.7.22
+
+#Used for generating notable release notes
+notableVersions=2.7.0, 2.6.1, 2.5.0, 2.4.0
```

Comment:
I owe the team a documentation how the new release will work. In short, for now:
 - the notableVersions list only influences the notable release notes generation
 - when you bump minor/major version new notable version is automatically included in version.properties and pushed to central / jcenter
 - above also happens when magic incantation is found in commit message